tools/pngtohpp.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+set -euo pipefail
+
+build_tool () {
```

Comment:
1. if you want to add c/c++ code add it properly to cmake as a target.
2. how is this tool documented and integrated?